test_scripts/Resumption/Handling_errors_from_HMI/009_restore_subscription_for_app2_after_fail_for_app1.lua

```diff
@@ -0,0 +1,90 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0190-resumption-data-error-handling.md
+--
+-- Description:
+-- In case:
+-- 1. Subscriptions for data_1 and data_2 are added by app1
+-- 2. Subscriptions for data_2 and data_3 are added by app2
+-- 3. Unexpected disconnect and reconnect are performed
+-- 4. App1 and app2 reregister with actual HashId
+-- 5. VehicleInfo.SubscribeVehicleData(data_1, data_2) related to app1 is sent from SDL to HMI during resumption
+-- 6. VehicleInfo.SubscribeVehicleData(data_2, data_3) request is not sent yet for app2
+-- 7. HMI responds with error resultCode VehicleInfo.SubscribeVehicleData(data_1, data_2) request
+-- 8. VehicleInfo.SubscribeVehicleData related to app2 is sent from SDL to HMI during resumption
+-- 9. HMI responds with success to remaining requests
+-- SDL does:
+-- 1. process unsuccess response from HMI
+-- 2. respond RegisterAppInterfaceResponse(success=true,result_code=RESUME_FAILED) to mobile application app1
+-- 3. restore all data for app2 and respond RegisterAppInterfaceResponse(success=true,result_code=SUCCESS)to mobile application app2
```

Comment:
Add additional step VehicleInfo.SubscribeVehicleData(data_2, data_3) request is sent to HMI

Reply:
Done in 3b6848c in line 20